cob_bringup/robots/cob4-2.xml

```diff
@@ -231,7 +229,6 @@
 	<group>
 		<machine name="cob4-2-h1" address="$(arg cob4-2-h1)" env-loader="$(find cob_bringup)/env.sh" default="true" timeout="30"/>
```

Comment:
should not be also commented? cob4-2 is the default setup of the cob4-1/cob4-2 robot, at least upstart and the command  cob-start are launching cob4-2 , if a pc machine can not be found the launch will die intermediately